lib/data/repositories/v1/account_repository.dart

```diff
@@ -75,6 +75,16 @@ class AccountRepository {
     }
   }
 
+  Future<void> updatePasscode(String passcode) async {
```

Comment:
This should be an `Either` of e.g. `Either<ApiError, boolean>` maybe? You can probably just return the either from `UpdateUser()`

Reply:
Now returns `Future<Either<ApiError, void>>`